src/test/java/org/gridsuite/filter/expertfilter/NumberExpertRuleTest.java

```diff
@@ -2383,4 +2388,204 @@ private static Stream<Arguments> provideArgumentsForStaticVarCompensatorTest() {
         );
     }
 
+    private static Stream<Arguments> provideArgumentsForHvdcLinesTest() {
+        HvdcLine hvdcLine = Mockito.mock(HvdcLine.class);
+        Mockito.when(hvdcLine.getType()).thenReturn(IdentifiableType.HVDC_LINE);
+        Mockito.when(hvdcLine.getR()).thenReturn(0.1);
+        Mockito.when(hvdcLine.getMaxP()).thenReturn(200.0);
+        Mockito.when(hvdcLine.getActivePowerSetpoint()).thenReturn(100.0);
+
+        // Terminal fields
+        Terminal terminal = Mockito.mock(Terminal.class);
+        VoltageLevel voltageLevel = Mockito.mock(VoltageLevel.class);
+        Mockito.when(voltageLevel.getNominalV()).thenReturn(13.0);
+        Mockito.when(terminal.getVoltageLevel()).thenReturn(voltageLevel);
+
+        // Converter Stations fields
+        HvdcConverterStation converterStation = Mockito.mock(HvdcConverterStation.class);
+        Mockito.when(converterStation.getTerminal()).thenReturn(terminal);
+        Mockito.when(hvdcLine.getConverterStation1()).thenReturn(converterStation);
+        Mockito.when(hvdcLine.getConverterStation2()).thenReturn(converterStation);
```

Comment:
In a real network, is it possible a hdvc line share the same converterStation and terminal?
If not it is better to separate in two stations so two terminals

Reply:
ok I'll change that